### DISCUSSION TITLE
How To Use Without Building? I'm Not Using Typescript...

### DISCUSSION THREAD

**Discussion by StevenLeonCooper** [vuejs/petite-vue#229] [Q&A]
I thought the whole point of petite-vue was to be able to use it to enhance existing sites and if you didn't want to build but the /src folder is full of Typescript files. I clone the repository and launch the root site in XAMPP but none of the examples work since the modules won't load. 

I just wanted something I could use on a vanilla JS/HTML app with no building or transpiling or extra steps at the beginning.

**Comment by sqllyw** [comment]
you asked two questions actually:

1) to use petite-view in your project:

    npm run build

    and you will see a dist folder, just link one of those files into your project, ignore the /src folder

2) to try the examples:

    npm install vite
    npm run dev
    open http://localhost:3000/ in a browser

    you will see Examples and Tests

**Reply by StevenLeonCooper** [reply]
Ah okay I cloned the whole repository and I just figured the examples were going to be built off an already "deployed" version and work out of the box. I'm really just wanting to try out Vue  in the simplest form possible so I can see if it's reasonable to try and teach it for a front-end web development course. What I'm using now is Vanilla JS with Mustache.js which is pretty bare bones.

**Reply by sqllyw** [reply]
actually i never cloned the whole repository when i started using petite-vue, finally did that when I want to trace some issues

**Comment by sqllyw** [comment]
also take a look at this for the errors you might encounter while running the examples:
https://github.com/vuejs/petite-vue/discussions/205

**Reply by jikkuatwork** [reply]
I ran the examples by referencing Petite Vue's CDN urls.

**Comment by sdesalas** [comment]
Some pre-compiled release files would be nice.

Happy to send a PR with the github action if you like.
